src/main/java/com/sprint/mission/discodeit/service/jcf/JCFMessageService.java

```diff
@@ -0,0 +1,54 @@
+package com.sprint.mission.discodeit.service.jcf;
+
+import com.sprint.mission.discodeit.entity.Message;
+import com.sprint.mission.discodeit.service.ChannelService;
+import com.sprint.mission.discodeit.service.MessageService;
+import com.sprint.mission.discodeit.service.UserService;
+
+import java.util.*;
+
+public class JCFMessageService implements MessageService {
+    private final Map<UUID, Message> data = new HashMap<>();
+    private final UserService userService;
+    private final ChannelService channelService;
+
+    public JCFMessageService(UserService userService, ChannelService channelService) {
+        this.userService = userService;
+        this.channelService = channelService;
+    }
+
+    @Override
+    public void createMessage(Message message) {
+        if(userService.getUser(message.getUser().getId()) == null) {
```

Comment:
UserService에 userId를 넘겨서 존재하는 유저인지 확인해도 괜찮을 것 같아요!  
```suggestion
        if(!userService.isExist(message.getUserId())) {
```